pubspec.yaml

```diff
@@ -12,6 +12,6 @@ dependencies:
   # the generated code. We could make the codegen ensure that it is listed in
   # the package pubspec instead.
   polymer_interop: ">=0.1.0+2 <0.2.0"
-  web_components: ">=0.11.2 <0.12.0"
+  web_components: ">=0.11.2 <0.12.3"
```

Comment:
I just went with the lowest version that'd unblock https://github.com/dart-lang/core-elements/pull/218 -- happy to increase to 0.13.0 if it's safe.